docs/root/intro/arch_overview/observability/tracing.rst

```diff
@@ -163,3 +160,60 @@ Tracing providers have varying level of support for getting and setting baggage:
 * Lightstep (and any OpenTelemetry-compliant tracer) can read/write baggage
 * Zipkin support is not yet implemented
 * X-Ray and OpenCensus don't support baggage
+
+Different types of span
```

Comment:
nit: can you run this text through ChatGPT and ask it to clean up the grammar/prose? I could do it, but that seems faster. :)

Reply:
let's me have a try. Basically, these text completed by the help of copilot. 🤣

Reply:
From my point, now it seems much better.

AI is pretty good tool for peopele who is not good at English (may be not just English).